run_integration_tests.sh

```
@@ -0,0 +1,94 @@
+    echo -e "${GREEN}========================================${NC}"
+else
+    echo -e "${RED}========================================${NC}"
+    echo -e "${RED} ❌ TESTS FAILED${NC}"
+    echo -e "${RED}========================================${NC}"
+fi
+
+exit $TEST_RESULT
+    echo -e "${RED}========================================${NC}"
+    echo -e "${RED} ❌ TESTS FALLARON${NC}"
+    echo -e "${RED}========================================${NC}"
+fi
```

Shellcheck (0.11.0)

[error] Parsing stopped here. Is this keyword correctly matched up?

(SC1089)